docker/Dockerfile.demo_vitis_ai

```diff
@@ -18,6 +18,8 @@
 # Main Vitis AI docker env
 FROM xilinx/vitis-ai:1.4.916
 
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 6AF7F09730B3F0A4
```

Comment:
Can you add a comment describing which key this is, why it needs to be added?  Searching for the key finds [this stackoverflow post](https://askubuntu.com/a/1390307), suggesting that it is related to kitware/cmake key, but it would be good to have that information in the file as well.

Reply:
Sorry, When I use this Dockerfile to build docker image, I met this error:

 **Err:7 https://apt.kitware.com/ubuntu bionic InRelease
The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 6AF7F09730B3F0A4**
So, I add this 
RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 6AF7F09730B3F0A4
in dockrfile, and solved this error.